client/cli/src/runner.rs

```diff
@@ -124,9 +124,12 @@ impl<C: SubstrateCli> Runner<C> {
 		let task_executor = move |fut, task_type| {
 			match task_type {
 				TaskType::Async => runtime_handle.spawn(fut).map(drop),
-				TaskType::Blocking =>
-					runtime_handle.spawn_blocking(move || futures::executor::block_on(fut))
-						.map(drop),
+				TaskType::Blocking => {
```

Comment:
Why do this change? As far as I can tell the previous version compiled fine.

Reply:
This was basically because of the panic that happens when you run `block_on` while a pool is already running. I had to use an alternative to `block_on` here so that we can `block_on` inside the keystore.